packages/site-client/package.json

```diff
@@ -23,7 +23,7 @@
       ]
     },
     "build:dev": {
-      "command": "find src -name \"*.ts\" | xargs esbuild --format=esm --target=es2022 --color --outdir=lib",
+      "command": "find src -name \"*.ts\" | xargs esbuild --color --outdir=lib",
```

Comment:
The default is `--target=esnext` -- that doesn't seem right, even for dev?

Reply:
We have this set in the tsconfig